package.json

```diff
@@ -15,10 +15,12 @@
     "@angular/common": "~10.2.0",
```

Comment:
Could you please make sure that all TS Lint issues are fixed in your project?

Reply:
fixed